clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -732,11 +614,8 @@ public void testUnsupportedCiphers() throws Exception {
 
         sslClientConfigs.put(SslConfigs.SSL_CIPHER_SUITES_CONFIG, Arrays.asList(cipherSuites[1]));
         createSelector(sslClientConfigs);
-        InetSocketAddress addr = new InetSocketAddress("localhost", server.port());
-        selector.connect(node, addr, BUFFER_SIZE, BUFFER_SIZE);
 
-        NetworkTestUtils.waitForChannelClose(selector, node, ChannelState.State.AUTHENTICATION_FAILED);
-        server.verifyAuthenticationMetrics(0, 1);
```

Comment:
Shall we keep the authentication metrics check?

Reply:
Fixed.